apps/meteor/app/lib/server/functions/saveUser.js

```diff
@@ -237,11 +237,16 @@ export async function validateUserEditing(userId, userData) {
 
 const handleBio = (updateUser, bio) => {
 	if (bio && bio.trim()) {
-		if (typeof bio !== 'string' || bio.length > MAX_BIO_LENGTH) {
+		if (typeof bio !== 'string') {
```

Comment:
It's kinda weird that we do a `trim` and then we check for the value to be an string tbh

Reply:
Agree. Removed this check from `bio` and `username` in this file

Reply:
For safety i think you can keep them, but will be better placed before trying to call `.trim()` :)